crates/aptos-rosetta/src/account.rs

```diff
@@ -172,7 +179,171 @@ async fn get_balances(
                                 currency: native_coin(),
                             })
                         }
-                    } /* TODO: Right now operator stake is not supported
+                    } else if account.is_pending_active_stake() {
+                        // For total pending active stake, collect all underlying staking contracts and combine
+                        let mut total_pending_active: Option<u64> = None;
+                        maybe_operators = Some(vec![]);
```

Comment:
What's the meaning of keeping the `maybe_operators`?

Reply:
i could be understanding this wrong, but it's looping through all staking contracts and totaling the balances by contract pool address. the `maybe_operators` is also one of the return values.